test/e2e/case38_install_operator_test.go

```diff
@@ -198,9 +210,9 @@ var _ = Describe("Testing OperatorPolicy", Ordered, Label("supports-hosted"), fu
 		}
 
 		DeferCleanup(func() {
-			utils.Kubectl("delete", "-n", testNamespace, "operatorpolicy", "--all")
-			utils.Kubectl(
-				"delete", "event", "--field-selector=involvedObject.name="+parentPolicyName, "-n", testNamespace,
+			utils.KubectlDelete("-n", testNamespace, "operatorpolicy", "--all")
```

Comment:
I'd keep this one as it was since we want to ensure the namespace has cleaned up before moving on to the next step (i.e. no --wait=false)

Reply:
Same comment applies to the rest of the operator policy tests.

Reply:
Updated. I've added a check so that `--wait` can be supplied to the function.